### ISSUE TITLE
DEFAULT_MX -- miss a "."

### ISSUE THREAD

**Issue by skhaen** [AlternC/AlternC#70]
In `/etc/alternc/local.sh`, the `DEFAULT_MX` MUST HAVE a dot at the end:

```
DEFAULT_MX="example.octopuce.fr."
```

and not

```
DEFAULT_MX="example.octopuce.fr"
```

Without this dot, the MX is something like `example.octopuce.fr.example.octopuce.fr`

**Comment by innoticFR** [comment]
The ./install/mysql.sql file includes the dot after the variable in the definition of defmx inside `domains_type`.

You should check your `domains_type` table.

   ./install/upgrades/1.0.0.sql:('defmx', 'Default mail server', 'NONE', '%SUB% IN MX 5 @@DEFAULT_MX@@.', 'vhost,url,ip,webmail,ipv6,cname,txt,defmx2',true, false, true, 'ADMIN'),
   ./install/mysql.sql:('defmx',  'Default mail server',        'NONE',      '%SUB% IN MX 5 @@DEFAULT_MX@@.',            'vhost,url,ip,ipv6,txt,defmx2',              true,     true,     true,  'ADMIN'),

**Comment by innoticFR** [comment]
This was fixed by 8485d8e20d08bac3924a151382ede0b60eacdf17
